test/e2e_node/jenkins/e2e-node-jenkins.sh

```diff
@@ -28,7 +28,16 @@ set -x
 
 . $1
 
+if [ "$INSTALL_GODEP" = true ] ; then
+  go get -u github.com/tools/godep
+  go get -u github.com/onsi/ginkgo/ginkgo
+  go get -u github.com/onsi/gomega
+fi
+
 make generated_files
+
+# TODO converge build steps with hack/build-go some day if possible.
+go generate test/e2e/framework/gobindata_util.go
```

Comment:
do the node e2e tests even use any of the `examples/` files? @pwittrock @timstclair